docker/master/Dockerfile

```diff
@@ -8,7 +9,9 @@ FROM debian:jessie
 RUN groupadd -r freeswitch --gid=999 && useradd -r -g freeswitch --uid=999 freeswitch
 
 # grab gosu for easy step-down from root
-RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
+RUN apt-get update && apt-get install -y --no-install-recommends dirmngr gnupg2 \
+    && gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
```

Comment:
Keep in mind that the key was changed. Also SignalWire token is required now.
https://freeswitch.org/confluence/display/FREESWITCH/Debian
Here is where you can find the key without auth.
https://keys.openpgp.org/search?q=655DA1341B5207915210AFE936B4249FA7B0FB03

Reply:
You can try this
```
gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys 655DA1341B5207915210AFE936B4249FA7B0FB03
```

Reply:
Thanks for reviewing this PR, Is there any alternative way for getting package without needing PAT? I think public open source application must does not need to dependent to specific user.

Reply:
@andywolk I get PAT from ARG in Dockerfile to does not mandatory expose private info in GitHub

Reply:
> Did I have to upload my PAT to this PR? @andywolk

Do not upload your PAT. 
Yes you can use `ARG` in the Dockerfile.